ArduPlane/quadplane.cpp

```diff
@@ -1763,6 +1763,8 @@ void QuadPlane::update(void)
     if (motors->armed()) {
         const bool motors_active = in_vtol_mode() || assisted_flight;
         if (motors_active && (motors->get_spool_state() != AP_Motors::SpoolState::SHUT_DOWN)) {
+            // log ANG at main loop rate
+            attitude_control->Write_Attitude();
```

Comment:
should not call then when use_fw_attitude_controllers() is true

Reply:
Fixed